programs/openbook-twap/src/lib.rs

```diff
@@ -233,34 +268,7 @@ pub mod openbook_twap {
             if let (Some(best_bid), Some(best_ask)) = (best_bid, best_ask) {
                 // we use average_ceil because (best_bid + best_ask) / 2 can overflow
                 let spot_price = best_bid.average_ceil(&best_ask) as u64; 
```

Comment:
do you think we could factor this logic out as well?

doesn't need to be in this PR, just curious